eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -358,7 +358,7 @@ function GetExistingTags($apiUrl) {
 }
 
 # Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
-function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha, $exitOnError = $True) {
+function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
```

Comment:
I'm fine with this for now but we should consider splitting this functionality up into a GetPackageMetadata and the validate tags functionality.